stock_available_to_promise_release/models/stock_move.py

```diff
@@ -82,7 +82,7 @@ def _previous_promised_qty_sql_lateral_where(self):
                 AND p_type.code = 'outgoing'
                 AND loc.parent_path LIKE ANY(%(location_paths)s)
                 AND (
-                    m.need_release = true
+                    COALESCE(m.need_release, False) = COALESCE(move.need_release, False)
```

Comment:
This SQL query should never be called on moves if the flag `need_release` is not `True`, so this shouldn't be necessary: https://github.com/OCA/wms/pull/106/files#diff-fca2e9865c2d75a9793dbe699bffdf976606d996b69d09cacd25660f2926e998R248-R254

Reply:
This line https://github.com/OCA/wms/pull/106/files#diff-fca2e9865c2d75a9793dbe699bffdf976606d996b69d09cacd25660f2926e998R208 should be changed to `for move in moves:`.

And in `_should_compute_ordered_available_to_promise`: https://github.com/OCA/wms/pull/106/files#diff-fca2e9865c2d75a9793dbe699bffdf976606d996b69d09cacd25660f2926e998R251, the filter on `need_release` be removed.

Reply:
Thanks for the review. I fixed thoses